pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -199,6 +215,17 @@ func New(
 
 	ctrl.featureGateAccess = featureGateAccess
 
+	ctrl.configInformerFactory = configInformerFactory
+
+	enabled, err := ctrl.sigstoreAPIEnabled()
+	if enabled {
+		if err := ctrl.addImagePolicyObservers(); err != nil {
+			klog.Infof("skipping error adding image policy observers: %v", err)
```

Comment:
```suggestion
			klog.Infof("error adding image policy observers: %v", err)
```
? Also in one other place.